pkg/arvo/web/testing.udon

```diff
@@ -1,58 +0,0 @@
-:-  ~[comments+&]
```

Comment:
As far as I know, this is the only documentation for `+test`. Not sure where it should go -- and it probably shouldn't go in `web/` -- but it should not be deleted.

Reply:
Added a new testing udon within the tests directory.